outputs/kafka.go

```diff
@@ -3,31 +3,119 @@ package outputs
 import (
 	"context"
 	"encoding/json"
+	"fmt"
 	"log"
+	"net"
+	"strings"
+	"time"
 
 	"github.com/DataDog/datadog-go/statsd"
 	"github.com/segmentio/kafka-go"
+	"github.com/segmentio/kafka-go/sasl/plain"
+	"github.com/segmentio/kafka-go/sasl/scram"
 
 	"github.com/falcosecurity/falcosidekick/types"
 )
 
 // NewKafkaClient returns a new output.Client for accessing the Apache Kafka.
 func NewKafkaClient(config *types.Configuration, stats *types.Statistics, promStats *types.PromStatistics, statsdClient, dogstatsdClient *statsd.Client) (*Client, error) {
 
+	transport := &kafka.Transport{
+		Dial: (&net.Dialer{
+			Timeout:   3 * time.Second,
```

Comment:
do we want to allow people to set this timeout?

Reply:
Probably, this value is the same as what is was prior to this change:
```go
var DefaultTransport RoundTripper = &Transport{
	Dial: (&net.Dialer{
		Timeout:   3 * time.Second,
		DualStack: true,
	}).DialContext,
}
```
I wasn't sure about going overboard on making every possible value configurable

Reply:
Ok, let's keep it like it is right now. Thanks